### ISSUE TITLE
Purpose of loss_fn()?

### ISSUE THREAD

**Issue by tarunbhatiaind** [abhishekkrthakur/bert-entity-extraction#8]
Hello Abhishek and my fellow NLP enthusiasts , i have a question, i read the documentation for BERTmodel and as far as i know it already returns crossentropy loss by default, i dont understand the reason of creating a loss_fn() for the purpose. Also, attention mask already handles the padded sequence so it allows to calculate loss only on the non padded tokens. Can someone let know why a separate  function is created here. Sorry if I am missing something :)
